packages/design-tokens/css/default-theme.css

```diff
@@ -1,302 +1,1696 @@
+  --kz-var-border-borderless-border-color-rgb-id: --kz-var-border-borderless-border-color-rgb;
   --kz-var-border-borderless-border-color-rgb-params: 0, 0, 0;
+  --kz-var-border-borderless-border-color-id: --kz-var-border-borderless-border-color;
   --kz-var-border-focus-ring-border-width: 2px;
+  --kz-var-border-focus-ring-border-width-id: --kz-var-border-focus-ring-border-width;
   --kz-var-border-focus-ring-border-radius: 10px;
+  --kz-var-border-focus-ring-border-radius-id: --kz-var-border-focus-ring-border-radius;
   --kz-var-border-focus-ring-border-style: solid;
+  --kz-var-border-focus-ring-border-style-id: --kz-var-border-focus-ring-border-style;
   --kz-var-color-wisteria-100: #f0f1f4;
+  --kz-var-color-wisteria-100-rgb: 240, 241, 244;
+  --kz-var-color-wisteria-100-rgb-id: --kz-var-color-wisteria-100-rgb;
   --kz-var-color-wisteria-100-rgb-params: 240, 241, 244;
+  --kz-var-color-wisteria-100-id: --kz-var-color-wisteria-100;
   --kz-var-color-wisteria-200: #e1e2ea;
+  --kz-var-color-wisteria-200-rgb: 225, 226, 234;
+  --kz-var-color-wisteria-200-rgb-id: --kz-var-color-wisteria-200-rgb;
   --kz-var-color-wisteria-200-rgb-params: 225, 226, 234;
+  --kz-var-color-wisteria-200-id: --kz-var-color-wisteria-200;
   --kz-var-color-wisteria-300: #c4c5d4;
+  --kz-var-color-wisteria-300-rgb: 196, 197, 212;
+  --kz-var-color-wisteria-300-rgb-id: --kz-var-color-wisteria-300-rgb;
   --kz-var-color-wisteria-300-rgb-params: 196, 197, 212;
+  --kz-var-color-wisteria-300-id: --kz-var-color-wisteria-300;
   --kz-var-color-wisteria-400: #a7a9c1;
+  --kz-var-color-wisteria-400-rgb: 167, 169, 193;
+  --kz-var-color-wisteria-400-rgb-id: --kz-var-color-wisteria-400-rgb;
   --kz-var-color-wisteria-400-rgb-params: 167, 169, 193;
+  --kz-var-color-wisteria-400-id: --kz-var-color-wisteria-400;
   --kz-var-color-wisteria-500: #898ba9;
+  --kz-var-color-wisteria-500-rgb: 137, 139, 169;
+  --kz-var-color-wisteria-500-rgb-id: --kz-var-color-wisteria-500-rgb;
   --kz-var-color-wisteria-500-rgb-params: 137, 139, 169;
+  --kz-var-color-wisteria-500-id: --kz-var-color-wisteria-500;
   --kz-var-color-wisteria-600: #6b6e94;
+  --kz-var-color-wisteria-600-rgb: 107, 110, 148;
+  --kz-var-color-wisteria-600-rgb-id: --kz-var-color-wisteria-600-rgb;
   --kz-var-color-wisteria-600-rgb-params: 107, 110, 148;
+  --kz-var-color-wisteria-600-id: --kz-var-color-wisteria-600;
   --kz-var-color-wisteria-700: #4b4d68;
+  --kz-var-color-wisteria-700-rgb: 75, 77, 104;
+  --kz-var-color-wisteria-700-rgb-id: --kz-var-color-wisteria-700-rgb;
   --kz-var-color-wisteria-700-rgb-params: 75, 77, 104;
+  --kz-var-color-wisteria-700-id: --kz-var-color-wisteria-700;
   --kz-var-color-wisteria-800: #35374a;
+  --kz-var-color-wisteria-800-rgb: 53, 55, 74;
+  --kz-var-color-wisteria-800-rgb-id: --kz-var-color-wisteria-800-rgb;
   --kz-var-color-wisteria-800-rgb-params: 53, 55, 74;
+  --kz-var-color-wisteria-800-id: --kz-var-color-wisteria-800;
+  --kz-var-color-purple-100: #f0f1f4;
```

Comment:
These `kz-var-color-xxx` tokens will need to be removed. I believe we should only use the new colour names without the prefix.

Reply:
ah yep good pick up!